### DISCUSSION TITLE
Data submission to Firefly III

### DISCUSSION THREAD

**Discussion by rasemaier** [firefly-iii/firefly-iii#8965] [Firefly III]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [ ] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [ ] ... there are no open or closed issues that are related to my problem
- [ ] ... it's [definitely a Firefly III issue, not me](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

Importer is not working.
I'm only using the first 4 columns

### Debug information

Example Data:
IBAN;Booked At;Text;Credit/Debit Amount;Balance;Valuta Date
CH0000000000000000000;21.05.2021;Überweisung TestTestTest;-50;96931.24;21.05.2021 00:00
CH0000000000000000000;25.05.2021;Überweisung TestTestTest;-16;96915.24;25.05.2021 00:00
CH0000000000000000000;25.05.2021;Überweisung TestTestTest;5114.65;102029.89;25.05.2021 00:00

Error(s) from the import process
Line #0: transactions.0.date: transactions.0.date Feld muss ausgefüllt sein. (original value: "")
Line #1: transactions.0.date: transactions.0.date Feld muss ausgefüllt sein. (original value: "")
Line #2: transactions.0.date: transactions.0.date Feld muss ausgefüllt sein. (original value: "")

Config:

{
    "version": 3,
    "source": "ff3-importer-1.5.2",
    "created_at": "2024-06-10T16:01:00+02:00",
    "date": "Y-m-d",
    "default_account": 1,
    "delimiter": "semicolon",
    "headers": true,
    "rules": true,
    "skip_form": true,
    "add_import_tag": true,
    "roles": [
        "account-iban",
        "date_book",
        "description",
        "amount",
        "_ignore",
        "_ignore"
    ],
    "do_mapping": [
        false,
        false,
        false,
        false,
        false,
        false
    ],
    "mapping": [],
    "duplicate_detection_method": "classic",
    "ignore_duplicate_lines": true,
    "unique_column_index": 0,
    "unique_column_type": "internal_reference",
    "flow": "file",
    "content_type": "csv",
    "custom_tag": "",
    "identifier": "0",
    "connection": "0",
    "ignore_spectre_categories": false,
    "grouped_transaction_handling": "",
    "use_entire_opposing_address": false,
    "map_all_data": true,
    "accounts": [],
    "date_range": "",
    "date_range_number": 30,
    "date_range_unit": "d",
    "date_not_before": "",
    "date_not_after": "",
    "nordigen_country": "",
    "nordigen_bank": "",
    "nordigen_requisitions": [],
    "nordigen_max_days": "90",
    "conversion": true,
    "ignore_duplicate_transactions": true
}

**Comment by JC5** [comment]
Select "transaction date", or "date", not "booking date".
